README.md

```diff
@@ -64,16 +64,16 @@ access fields otherwise available through `proc_get_status()`.
 ### Stream Properties
 
 Once a process is started, its I/O streams will be constructed as instances of
-`React\Stream\Stream`. Before `start()` is called, these properties are `null`.
+`React\Stream\ReadableResourceStream` as `React\Stream\WritableResourceStream`. Before `start()` is called, these properties are `null`.
```

Comment:
as -> and

Also, should we use `ReadableStreamInterface` and `WritableStreamInterface` instead of the concrete implementations?